Regression/Checksum/benchmarks_json/averaged_galilean_3d_psatd.json

```diff
@@ -1,35 +1,35 @@
 {
   "electrons": {
-    "particle_cpu": 0.0,
-    "particle_id": 536887296.0,
-    "particle_momentum_x": 7.266679876000112e-22,
-    "particle_momentum_y": 7.383489173934850e-22,
-    "particle_momentum_z": 8.903836199717060e-17,
-    "particle_position_x": 8.192002938829830e+04,
-    "particle_position_y": 8.191963682799038e+04,
-    "particle_position_z": 4.859842246411671e+06,
+    "particle_cpu": 16384.0,
+    "particle_id": 268451840.0,
```

Comment:
@atmyers Do you happen to know why these two values would change so much, going from 1 MPI process and 1 grid to 2 MPI processes and 2 grids? The same happens for `ions` and for the other tests as well (except for `galilean_3d_psatd_current_correction` where the domain decomposition was not modified).

Reply:
That is normal: we split the sauce of possible IDs over MPI ranks. Safe to ignore when changing decomposition